project/Settings.scala

```diff
@@ -65,7 +69,7 @@ object Settings {
     commonBuildSettings ++ commonAssemblySettings ++ commonTestSettings ++ List(
       organization := "org.broadinstitute.dsde.workbench",
       scalaVersion := "2.13.10",
-      resolvers ++= commonResolvers,
+      resolvers := proxyResolvers ++: resolvers.value ++: commonResolvers,
```

Comment:
Not sure where resolvers is being defined, and I'm not sure what the `:=` operator does.  Looking at the scaldoc I couldn't find one on List.  So as long as the operators are right here

Reply:
Its what's in Rawls, so I'm assuming its correct 🤷‍♂️